lxd/metrics/types.go

```diff
@@ -16,8 +16,12 @@ type MetricSet struct {
 type MetricType int
 
 const (
+	// APICompletedRequests represents the total number completed requests
+	APICompletedRequests MetricType = iota
+	// APIOngoingRequests represents the number of requests currently being handled.
+	APIOngoingRequests MetricType = iota
 	// CPUs represents the total number of effective CPUs.
-	CPUs MetricType = iota
```

Comment:
this change isnt relevant to the commit

Reply:
Shouldn't this be includede? This line was just moved down because the new metrics were added before, and the `iota` definition is only needed on the first item.

Reply:
Oh right, yes ofc.